run_test.py

```diff
@@ -166,7 +166,7 @@
             'cudnn': 'cudnn7-cuda91',
             'nccl': 'nccl2.1-cuda91',
             'requires': [
-                'setuptools', 'cython==0.26.1', 'numpy<1.10', 'scipy<0.19',
+                'setuptools', 'cython==0.28.2', 'numpy<1.10', 'scipy<0.19',
```

Comment:
`No matching distribution found for cython==0.28.2`

Reply:
Fixed!